hack/run-image-scan.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Please add below:

* licensing boilerplate: https://github.com/kubermatic/kubermatic/blob/2f8eff72af97022d2f446c8950677f799fcfd437/hack/coverage.sh#L3-L15
* documentation for this script describing what's its purpose: https://github.com/kubermatic/kubermatic/blob/2f8eff72af97022d2f446c8950677f799fcfd437/hack/coverage.sh#L17-L21

Reply:
Done